protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -747,9 +749,19 @@ return network.registerProtocol('wireguard', {
 
 				var ips = [ '0.0.0.0/0', '::/0' ];
 
+				var dns = [ '1.1.1.1', '1.0.0.1' ];
```

Comment:
I would not set a default value for the DNS server.

Reply:
Done.